.travis.yml

```diff
@@ -0,0 +1,7 @@
+sudo: false
+language: node_js
+node_js:
+  - '4'
+  - '6'
+  - '7'
```

Comment:
I recommend using the `'stable'` keyword instead of specifying non-LTS versions so it's one less thing to maintain as time goes on.